### PR TITLE
Add a new target HolybroG4 DroneCAN Airspeed

### PR DESCRIPTION
**Pull request by jamming**
Holybro DroneCAN  airspeed meter is an industrial-grade differential pressure measurement module. It use AP_Periph firmware.
The DLVR airspeed sensor using dual-wafer technology has the characteristics of high accuracy and good linearity.It supports up to 226.8KM/h airspeed measurement, meeting the needs of most UAV applications.

### CODE REVIEW

File: libraries/AP_HAL_ChibiOS/hwdef/HolybroG4_Airspeed/hwdef-bl.dat [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+FLASH_BOOTLOADER_LOAD_KB 32
+
+# reserve some space for params
+APP_START_OFFSET_KB 4
+
+# 512k flash part
+FLASH_SIZE_KB 512
+
+# board ID for firmware load
+APJ_BOARD_ID 5405
```
Comment (peterbarker) [inline libraries/AP_HAL_ChibiOS/hwdef/HolybroG4_Airspeed/hwdef-bl.dat:16]: Needs a patch to board_types.txt

**Review by peterbarker — CHANGES_REQUESTED** [review]
Please run `./Tools/gittools/git-subsystems-split` to split and rename the commits properly.

File: libraries/AP_HAL_ChibiOS/hwdef/HolybroG4_Airspeed/hwdef-bl.dat (hunk 1 above)
Comment (jamming) [reply libraries/AP_HAL_ChibiOS/hwdef/HolybroG4_Airspeed/hwdef-bl.dat:16]: "AP_HW_Holybro-ASP-G4                 5405"  has been added to file  board_types.txt.
Thank you very much!

**Comment by peterbarker** [comment]
Replaces by https://github.com/ArduPilot/ardupilot/pull/27888

**Pull request closed without merging**